apps/studio/components/grid/components/header/filter/FilterPopover.tsx

```diff
@@ -131,10 +138,14 @@ const FilterOverlay = ({ table, filters: filtersFromUrl, setParams }: FilterOver
       </div>
       <Popover.Separator />
```

Comment:
Can you add a separator into the shadcn/popover?, can copy what is used in shadcn/Dropdown

<PopoverSeperator/>

Reply:
https://github.com/supabase/supabase/blob/02c0d75dfbe5af5260376a009bbf14e675d39b2d/packages/ui/src/components/shadcn/ui/dropdown-menu.tsx#L152-L162

Reply:
oh nice, the links show previews now ❤️

Reply:
actually your probably better copying [this one](https://github.com/supabase/supabase/blob/master/packages/ui/src/components/shadcn/ui/dialog.tsx/#L200-L206)

Reply:
https://github.com/supabase/supabase/blob/master/packages/ui/src/components/shadcn/ui/dialog.tsx/#L200-L206

Reply:
gotcha!